pkg/cluster/k8sres.go

```diff
@@ -1041,6 +1041,9 @@ func (c *Cluster) generateSpiloPodEnvVars(
 
 	envVars = appendEnvVars(envVars, opConfigEnvVars...)
 
+	// add variable containing the UID of the cluster; usefull for backup paths
+	envVars = append(envVars, v1.EnvVar{Name: "CLUSTER_UID", Value: getBucketScopeSuffix(string(uid))})
```

Comment:
@mkrueger92  probably you can set original `string(uid)` value to avoid confusion with `CLUSTER_ID` name, as `getBucketScopeSuffix(...)` returns `uid` combined with `/` prepared for the `<>_SUFFIX` variables.

Reply:
Good point. I will prepare a new commit.